blues/moves.py

```diff
@@ -177,70 +178,304 @@ def move(self, context):
                  single_move.move(context)
 
 
-class MoveEngine(object):
-    """MoveEngine provides perturbation functions for the context during the NCMC
-    simulation.
-    Ex.
-        from blues.ncmc import MoveEngine
-        probabilities = [0.25, 0.75]
-        #Where move is a list of two Move objects
-        mover = MoveEngine(move, probabilities)
-        #Get the dictionary of proposed moves
-        mover.moves
-    """
-    def __init__(self, moves, probabilities=None):
-        """Initialize the MovePropsal object that contains functions to perturb
-        the context in the NCMC simulation.
-        Parameters
-        ----------
-        moves : blues.ncmc.Model object or list of n blues.ncmc.Model-like objects
-            Specifies the possible moves to be performed.
+class SmartDartMove(RandomLigandRotationMove):
+    def __init__(self, structure, basis_particles, dart_size=0.2*unit.nanometers, resname='LIG'):
```

Comment:
Needs doc string saying what the initialization arguments are. For example, the `dart_size` and `basis_particles` arguments are fairly critical and the former does not seem to be described below. Because there was no doc string here I just read past it and kept wondering, "but wait, where is the dart size set?"